added "UPDATE STATISTICS" statement

ssuvorov-fls opened the pull request.

ssuvorov-fls committed:
added "UPDATE STATISTICS" statement

konstjar commented:
This PR is required to handle Impala memory issues. The same change were added in (circle-be and SQLRender packages)

jreps commented:
Thanks for the fix.  I noticed this pull require made many changes to the sql scripts (I think due to extra EOL characters in addition to adding UPDATE STATISTICS).  It is hard to review all the edits in the current pull request.  Is it possible to edit the sql scripts in a way that only UPDATE STATISTICS is added (no other edits) - so I can quickly check that is the only change and then pull the requests?

ssuvorov-fls committed:
fixed line ending

ssuvorov-fls committed:
fixed line ending

ssuvorov-fls committed:
fixed line ending

ssuvorov-fls committed:
fixed line ending

ssuvorov-fls committed:
fixed line ending

konstjar commented:
@jreps Thank you for checking.
It looks like most of initial SQL files in master branch have strange "cr" as EOL. You can see it as red ^M

![image](https://user-images.githubusercontent.com/4144173/79893470-6c4de000-8404-11ea-9e0f-db001fce0e51.png)

As a result when we clone master repo git transforms these EOL into additional line break.
Is it possible to fix it in master branch?

ssuvorov-fls committed:
Revert "added "UPDATE STATISTICS" statement"

This reverts commit 836874f1

ssuvorov-fls committed:
backmerge ohdsi to odysseusin (#2)

* updated protocol w covid val cohorts

* updaing the analysisId for correct saving

- fixed a bug where the analysisId of the extra models was incorrect

* updated protocol w RW as contact

* Update README.md

* added simple model validations

* updated codeToRun added simple wrapper to main

* adding cdmDatabaseName

* Add cdmDatabaseName

* adding initial single script

- still need to write script to move and rename zip files into exportAll folder

* updated for simple model Q1 in Q2

* Update codeToRun.R

* fixed packagResults bug

* fixed outcomeId issue

* Update Main.R

changing validationResults.rds to validationResult.rds

* Update PackageResults.R

changing validationResults.rds to validationResult.rds

* Update helpers.R

fixing date bug from yyyy-mm-dd to yyyymmdd

* Update helpers.R

changing dates from yyyy-mm-dd to yyyymmdd

* Update Main.R

changing years from yyyy-mm-dd to yyyymmdd

* Update README.md

* Update readme.md

* Update readme.md

* Update readme.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update readme.md

* Update readme.md

* Update readme.md

* Update readme.md

* Update readme.md

* Update README.md

* Update CodeToRun.R

adding packaging code

* Update README.md

* Update README.md

* Update README.md

* Update readme.md

* Update readme.md

* Update readme.md

* Revert "fixed outcomeId issue"

This reverts commit d856c8196a2be7851136772cca772602192ed6a2.

* Update README.md

* updated codeToRun added pneumonia

* updated readme with simpleScore

* fixed date format issue

* Update CodeToRun.R

* Update CodeToRun.R

* Update CodeToRun.R

- fixed packaging bug

* Update README.md

* Update README.md

* fixed diabetes cohort

* removing odd inpatient restriction from diabetes definition

* reducing sql names

* chaning names in settings

- changing names to match reduced sql names

* Update README.md

Co-authored-by: RossDWilliams <r.williams@erasmusmc.nl>
Co-authored-by: Jenna Reps <jreps@its.jnj.com>
Co-authored-by: Ross D. Williams <34747165+rossdwilliams@users.noreply.github.com>
Co-authored-by: Peter Rijnbeek <p.rijnbeek@erasmusmc.nl>
Co-authored-by: Martijn Schuemie <schuemie@ohdsi.org>

ssuvorov-fls committed:
Merge remote-tracking branch 'origin/master' into add_update_statistics_statement

# Conflicts:
#	CovidSimpleModels/inst/sql/sql_server/Persons with respiratory failure or insufficiency.sql
#	CovidSimpleModels/inst/sql/sql_server/[Covid v1] persons with cancer.sql

ssuvorov-fls committed:
merge conflicts
